src/contracts/core/DelegationManager.sol

```diff
@@ -602,8 +608,13 @@ contract DelegationManager is
         });
 
         // based on total magnitude, update operators delegatedShares
-        uint256 delegatedShares = DelegatedShares.unwrap(WithdrawableShares.wrap(addedShares).toDelegatedShares(totalMagnitude));
-        operatorDelegatedShares[operator][strategy] += delegatedShares;
+        DelegatedShares delegatedShares = addedShares.toDelegatedShares(totalMagnitude);
+
+        // forgefmt: disable-next-line
+        operatorDelegatedShares[operator][strategy] = 
+            operatorDelegatedShares[operator][strategy]
+                .add(delegatedShares.unwrap())
+                .wrapDelegated();
```

Comment:
why do we wrap here?

Reply:
operatorDelegatedShares mapping returns type DelegatedShares

Reply:
shouldn't .add return DelegatedShares?